build.sbt

```
@@ -19,6 +19,54 @@ lazy val nameSettings: Seq[Setting[_]] = Seq(
   name := projectName(thisProject.value) // Maven <name>
 )
 
+lazy val disabledDocsSettings: Seq[Setting[_]] = Def.settings(
+  sources in (Compile, doc) := Nil
+)
+
+lazy val docsSettings: Seq[Setting[_]] = {
+  val javaDocBaseURL: String = "http://docs.oracle.com/javase/8/docs/api/"
+  // partially ported from Scala.js
+  Def.settings(
+    autoAPIMappings := true,
+    exportJars := true,                                    // required so ScalaDoc linking works
+    scalacOptions in (Compile, doc) -= "-Xfatal-warnings", // needed only for 2.11 docs to mitigate not found java.lang members
```

Can we try without this? It might work in this build.

---

As pointed in a comment there is a single error existing only in 2.11 which caused adding this option. We can remove it if/when we would drop 2.11 support. 
```
[error] nativelib/src/main/scala/scala/scalanative/reflect/Reflect.scala:19:3: Could not find any member to link for "java.lang.InstantiationException".
[error]   /** Instantiates a new instance of this class using the zero-argument
[error]   ^
[error] one error found
```
I've tried to put the usage of this class inside commont in double-quotes but it does not fix the problem

---

If it's only in 2.11, we should add the second only in 2.11, then:
```suggestion
    scalacOptions in (Compile, doc) := {
      val prev = (scalacOptions in (Compile, doc)).value
      if (scalaVersion.value.startsWith("2.11.") prev.filter(_ != "-Xfatal-warnings")
      else prev
    }
```